src/Uno.UI.FluentTheme/Resources/Version1/PriorityDefault/FlipView_themeresources_v1.xaml

```diff
@@ -365,8 +365,4 @@
         </Setter>
     </Style>
 
-	<!-- UNO TODO Currently only style using native flip view control is supported on Android and iOS-->
-	<wasm:Style TargetType="FlipView" BasedOn="{StaticResource DefaultFlipViewStyle}" />
-	<skia:Style TargetType="FlipView" BasedOn="{StaticResource DefaultFlipViewStyle}" />
```

Comment:
Why did you remove this ?

Reply:
Mistake, will revert the V1 changes altogether